app/domain/authentication/authn_k8s/validate_pod_request.rb

```diff
@@ -5,14 +5,18 @@
 module Authentication
   module AuthnK8s
 
+    AUTHENTICATION_CONTAINER_NAME_ANNOTATION = "authentication-container-name"
```

Comment:
inject_client_cert.rb has the following comment

```

# This code is implemented similarly also in application_identity.rb
# We have it here too as we need the container name for the injection
# and it simplifies the code to have this specific 2 methods duplicated
# rather than passing around the ApplicationIdentity object
```
I think we should also update Application Identity here

Reply:
done, thanks